src/main/java/org/littletonrobotics/urcl/URCL.java

```diff
@@ -56,10 +63,31 @@ public static void start() {
   /**
    * Start capturing data from REV motor controllers to NetworkTables. This method
    * should only be called once.
-   * 
+   *
+   * @param withNT Whether or not to run with NetworkTables.
+   */
+  public static void start(boolean withNT) {
+    start(Map.of(), withNT);
+  }
+
+  /**
+   * Start capturing data from REV motor controllers to NetworkTables. This method
+   * should only be called once.
+   *
    * @param aliases The set of aliases mapping CAN IDs to names.
    */
   public static void start(Map<Integer, String> aliases) {
+    start(aliases, true);
+  }
+
+  /**
+   * Start capturing data from REV motor controllers to NetworkTables. This method
+   * should only be called once.
+   *
+   * @param aliases The set of aliases mapping CAN IDs to names.
+   * @param withNT Whether or not to run with NetworkTables.
+   */
+  public static void start(Map<Integer, String> aliases, boolean withNT) {
```

Comment:
Instead of a boolean API, would it be clearer if we had two methods instead?

Reply:
The `DriverStation` API accepts a `DataLog` object rather than just pulling from `DataLogManager`. Rather than the boolean, maybe have versions of that method that accept a `DataLog` and versions that don't (and if a log is provided, don't log to NT)?